README.md

```diff
@@ -62,6 +62,18 @@ $ ant -Dcluster.config=platform
 
 **Note:** You can also use `php`, `enterprise`, etc. See the [cluster.properties](https://github.com/apache/incubator-netbeans/blob/master/nbbuild/cluster.properties) file.
 
+#### Building Windows Launchers
+
+As of [NETBEANS-1145](https://issues.apache.org/jira/browse/NETBEANS-1145), the Windows Launchers can be built adding ```do.build.windows.launchers=true``` property to the build process.
+```
+$ ant -Ddo.build.windows.launchers=true
+```
+
+##### Software Requirement to Build Windows Launchers on Ubuntu:
+```
+sudo apt install gcc g++ cmake gdb qt4-qmake g+-mingw-w64-x86-64 g+-mingw mingw-w64 openjdk-8-jdk 
```

Comment:
I wonder if we want to say something along the lines of "MinGW is used to compile the Windows launchers. The following command line may install it on Ubuntu:"

The list of packages may? be unnecessarily broad here? "apt install make mingw-w64" might suffice, although I didn't try. (OTOH, requiring more packages is surely better than requiring too few.)

Reply:
I may retest the requirements on a LXC Xenial container later today. Honestly I was provided with this set of packages, and I asked the Apache infra to have them installed.